manifests/t/Tencent/WeCom/4.0.0.6023/Tencent.WeCom.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Tencent.WeCom
+PackageVersion: 4.0.0.6023
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://dldir1.qq.com/wework/work_weixin/WeCom_4.0.0.6023.exe
+  InstallerSha256: 8E51E3FDAB324A3F38EE51EB93A73E139C919A907E1C11C4D9D626F4091E84DE

```

```suggestion
  InstallerSha256: 8E51E3FDAB324A3F38EE51EB93A73E139C919A907E1C11C4D9D626F4091E84DE
  ProductCode: 企业微信
  Scope: machine
```
Adds `ProductCode` & `Scope`.